cv32e40s/regress/cv32e40s_hello_world.yaml

```diff
@@ -1,5 +1,5 @@
-# YAML file to specify the ci_check regression testlist.
-name: cv32e40s_ci_check
```

Comment:
This seems to just be copy-paste mistakes in the "name" field so I just fixed them to match the file name.